exp/agents/agentExecutor/agentExecutor.go

```diff
@@ -0,0 +1,7 @@
+package agentExecutor
```

Comment:
go files and package names should only consist of lowercase characters, see https://github.com/uber-go/guide/blob/master/style.md#package-names